docs/src/main/asciidoc/_elytron/components/Filesystem_Security_Realm.adoc

```diff
@@ -323,4 +324,26 @@ realm-name:realm-two
 
 There are other options that can be specified for converting a filesystem realm, the details of which can be found using the `./bin/elytron-tool.sh filesystem-realm-integrity -h` command. 
 
+== Updating WildFly Elytron Tool Generated CLI Scripts for Domain Mode
+As mentioned above, some of the WildFly Elytron Tool commands create a CLI script that can be used to add resources to
+the Elytron subsystem. Some examples of these CLI scripts include:
+
+. The <FILESYSTEM-REALM-NAME>.cli script generated when the `filesystem-realm-encrypt` function with the WildFly Elytron Tool is used to encrypt an existing filesystem realm.
+. The <FILESYSTEM-REALM-NAME>.cli script generated when the `filesystem-realm-inregrity` function with the WildFly Elytron Tool is used to add integrity checking to existing filesystem realms.
+. The <FILESYSTEM-REALM-NAME>.sh script generated when the `filesystem-realm` function with the WildFly Elytron Tool is used to create a filesystem realm from a legacy properties file.
+
+However, that CLI script can only be used directly if the server is running in Standalone mode. For Domain mode
+the script needs to be updated. The steps for updating the script are as follows:
+
+* Update the .cli file contents to add a /profile=<PROFILE_NAME> in front of all occurrences of the
+/subsystem=... strings as found in the file (so in front of /subsystem=elytron...); where <PROFILE_NAME> is the profile
+name that you are using in your WildFly domain.xml (Hence, <PROFILE_NAME> can be either default, ha, full, full-ha, or
+any other profile-name defined and used in the domain.xml.)
+
+* Save and Exit the file editor.
+
+Now you can use the CLI script to update server configurations in domain mode.
```

Comment:
s/configurations/configuration

Reply:
Shouldn't this be plural? My understanding is that in domain mode we have multiple servers running as a group. 
https://github.com/wildfly/wildfly/pull/17413#discussion_r1409427921

Reply:
Ah, yes, I missed that sorry. What you have is correct.

Reply:
No problem! 
I will push the changes for the other comments. 
Thank you!